source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -126,172 +130,137 @@ used at the data preparation and data processing stages for the following:
+table:
 
-- The :capterm:`data type` of each feature (e.g. :code:`float` or
-  :code:`double`).
-
-.. note::
-  Metadata can contain both compile-time and run-time information. For example,
-  basic compile-time metadata is the type of a dataset - whether it is a
-  particular :txtref:`data-source` or a :txtref:`table`. Run-time information
-  can contain the :capterm:`feature` types and :capterm:`data types <data type>`
-  of a dataset.
+1. The `data types <Data type>` of the columns.
 
-.. _table-builder:
+2. The logical types of data in the columns:
+   :capterm:`nominal <Nominal feature>`, :capterm:`ordinal <Ordinal feature>`,
+   :capterm:`interval <Interval feature>`, or :capterm:`ratio <Ratio feature>`.
+   Information about logical types is mostly relevant in cases when table
+   columns represent dataset features, but it is also applicable in other cases
+   as well, for example, when a table represents a matrix.
 
-Table builder
--------------
+Only the properties of data that do not affect table concept definition shall be
+the part of metadata concept.
 
-A table :capterm:`builder` is a concept that is associated with a particular
-:txtref:`table` type and is used at the data preparation and data processing
-stages for:
+.. warning::
+  While extending the table concept, specification implementer shall distinguish
+  whether a new property they are adding is a property of a particular ``table``
+  sub-type or a property of table metadata.
 
-- Iterative construction of a :txtref:`table` from another :txtref:`<table>s` or
-  a different in-memory :txtref:`dataset` representations.
-
-- Construction of a :txtref:`table` from different entities that hold blocks of
-  data, such as arrays, pointers to the memory, external entities.
-
-- Changing dataset values. Since :txtref:`table` is an
-  :capterm:`immutable <immutability>` dataset, a builder provides the ability to
-  change the values in a dataset under construction.
-
-- Encapsulating construction process of a :txtref:`table`. This is used to hide the
-  implementation details as they are irrelevant for users. This also allow to
-  select the most appropriate table implementation for each particular case.
-
-- Providing additional information on how to create a :txtref:`table` inside an
-  algorithm for :txtref:`<result>s`. This information includes metadata, memory
-  allocators that shall be used, or even a particular table implementation.
-
-For details, see :txtref:`table-builders` section.
+  For example, :capterm:`data layout` and :capterm:`data format` are properties
+  of table objects since they affect the structure of a table, its contract, and
+  behavior.
 
 .. _accessor:
 
 Accessor
 --------
 
-Accessor is a concept that defines a single way to get the data from an
-in-memory numerical :txtref:`dataset`. It allows:
+Accessor is a concept that defines a single way to extract the data from a
+:txtref:`table`. It allows to:
```

Comment:
minor: stylistic part - in this case we have 'to' in this line while in the rest cases above we have 'to' in each bullet. Probable, makes sense to align the rest with this case